deploy/lib/server_config.rb

```diff
@@ -728,6 +734,10 @@ def bootstrap
   end
 
   def wipe
+
+    @ml_username = @properties['ml.bootstrap-user'] || @properties['ml.user']
+    @ml_password = @properties['ml.bootstrap-password'] || @properties['ml.password']
```

Comment:
I think password selection needs to be a little more involved. 

    if ml.bootstrap-user, then use ml.bootstrap-password
      use it even if it's empty, which will result in calling password_prompt
    if !ml.bootstrap-user, then fall back to ml.password

Likewise for deploy-password below. This will handle the situation where some other password is used with the bootstrap user (unlikely, but quite possible).